tests/unit/conftest.py

```diff
@@ -26,12 +27,120 @@ def client():
         with TestClient(main.app) as c:
             yield c
 
-
 @pytest.fixture
 def basic_algorithm():
-    return utils.read_json_from_assets("linking", "basic_algorithm.json")["algorithm"]
-
+    basic_algo_pass1 = models.AlgorithmPass(
+        id=1, 
+        algorithm_id=1, 
+        blocking_keys=[1,2,3], 
```

Comment:
I was thinking we'd store the actual names here of the Blocking Keys for readability sake.  It adds to the storage, but this table is going to be small.  For example

```
blocking_keys=["BIRTHDATE", "SEX", "FIRST_NAME"]
```

Reply:
I could make that change. I would just need to change the `AlgorithmPass` model to store a `list[str]` for `blocking_keys`